numpy/random/tests/test_extending.py

```diff
@@ -44,6 +46,7 @@
         cython = None
 
 
+@pytest.mark.xfail(IS_WINDOWS, reason="Wheel builds are missing npyrandom.lib")
```

Comment:
I think this should be investigated in #23981

Reply:
It's already listed there